### ISSUE TITLE
the lr is always the same

### ISSUE THREAD

**Issue by github163sl** [zylo117/Yet-Another-EfficientDet-Pytorch#558]
Thank you for your implement for EfficientDet!
When I train only one class ,the lr is always the same.
Can you help me?Thank u

**Comment by zylo117** [comment]
learning rate will remain unchanged unless it's not training well

**Comment by github163sl** [comment]
Thank U!
